test/util/malicious/out_of_order_prepare.go

```diff
@@ -46,11 +46,23 @@ func (a *App) OutOfOrderPrepareProposal(req *abci.RequestPrepareProposal) (*abci
 		a.GovParamFilters(),
 	)
 
-	txs := app.FilterTxs(a.Logger(), sdkCtx, handler, a.GetTxConfig(), req.Txs)
+	fb, err := app.NewFilteredBuilder(
+		a.Logger(),
+		sdkCtx,
+		handler,
+		a.GetEncodingConfig().TxConfig,
+		a.MaxEffectiveSquareSize(sdkCtx),
+		appconsts.SubtreeRootThreshold,
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	txs := fb.Fill(req.Txs)
 
 	// build the square from the set of valid and prioritised transactions.
 	// The txs returned are the ones used in the square and block
-	dataSquare, txs, err := Build(txs, appconsts.LatestVersion, a.MaxEffectiveSquareSize(sdkCtx), OutOfOrderExport)
+	dataSquare, err := fb.Build()
```

Comment:
really hope we can just delete this package as maintaining two copies of things is cursed #4990